### ISSUE TITLE
open best checkpoint

### ISSUE THREAD

**Issue by Thomas-pku** [filaPro/unidet3d#12]
I think your work is great，Can you open source your best performing weights?

**Comment by filaPro** [comment]
Unfortunately, there is no single best checkpoint due to randomness in training and inference. However, the provided checkpoint achieves almost all *maximum* metrics reported in the paper, and even outperforming some of them.

**Comment by Thomas-pku** [comment]
Where can I find the checkpoint files you provided? I used the two checkpoints you uploaded, and all indicators were almost 0.

**Comment by filaPro** [comment]
This looks like some bug with your installation. The provided checkpoint should reproduce metrics from our readme.

**Comment by Thomas-pku** [comment]
If I use the checkpoint you provided, and test with only a single scannet, will I reproduce your effect?

**Comment by filaPro** [comment]
Yes should be fine. Can you privide the complete log of your test run?

**Comment by Thomas-pku** [comment]
I run 'python tools/test.py configs/unidet3d_1xb8_scannet.py /home/lsh/unidet3d.pth'
[20241223_013616.log](https://github.com/user-attachments/files/18222727/20241223_013616.log)

**Comment by Thomas-pku** [comment]
If I run /configs/unidet3d_1xb8_scannet_s3dis_multiscan_3rscan_scannetpp_arkitscenes.py /home/lsh/unidet3d.pth'，and comment out other test sets.It does reproduce the original effect. Can you tell me why?

**Comment by filaPro** [comment]
Looks like package versions (torch, spconv, cuda) don't match our Dockerfile. Can you try with exact same versions or run it in docker?

**Comment by bradduy** [comment]
Hi @Thomas-pku , have you found where the problem is? I'm facing the same issue with scores nearly 0.

**Comment by filaPro** [comment]
Our checkpoint in corresponding to the 6-dataset config. Please use 6-dataset config and comment non-necessary datasets from validation.
